backends/smv/smv.cc

```diff
@@ -358,15 +358,16 @@ struct SmvWorker
 				continue;
 			}
 
-			if (cell->type.in(ID($div), ID($mod)))
+			// SMV has a "mod" operator, but its semantics don't seem to be well-defined - to be safe, don't generate it at all
+			if (cell->type.in(ID($div)/*, ID($mod), ID($modfloor)*/))
```

Comment:
@cliffordwolf: you implemented this back in 8a86162ae9a4e2939af162edfc6fa281931f786e, do you have any further information on the exact semantics of SMV's "mod"?

Reply:
Good question. I have no idea.